src/pages/CatIndex.js

```diff
@@ -1,9 +1,95 @@
 import React, { Component } from 'react'
+import { Card, CardImg, CardText, CardBody, CardTitle, CardSubtitle, Button } from 'reactstrap';
+import { NavLink } from 'react-router-dom'
+// import { Carousel, CarouselItem, CarouselControl, CarouselIndicators, CarouselCaption } from 'reactstrap';
+// import Carousel from 'react-multi-carousel';
 
 export class CatIndex extends Component {
+  // constructor(props) {
```

Comment:
There is a lot of commented-out code on this page.  It is best practice to remove any unused code.  It also looks like you have multiple exports on each file, usually, you will only want to have one export for the class.